### ISSUE TITLE
Jupyter Notebook with exogenous data examples

### ISSUE THREAD

**Issue by iKintosh** [tinkoff-ai/etna#353]
### 🚀 Feature Request

Add Jupyter Notebook with examples on how to use exogenous data

### Motivation

It would be nice to have examples for exogenous data and how easy to use it with etna.

### Proposal

1. Find open dataset with exogenous data (covid for example)
2. Describe how additional data and regressors are different (both exogenous data)
3. Show how to make additional data into regressor and why it is necessary 
    - Show what to do if exogenous series is common for all series
    - Show what to do if each series has its own exogenous series
4. Show that models work with exogenous data automatically 

### Checklist

- [x] I discussed this issue with ETNA Team

**Comment by julia-shenshina** [comment]
is blocked by all the issues from Regressors 2.0 milestone

**Comment by alex-hse-repository** [comment]
We can use dataset from [this](https://www.kaggle.com/c/tabular-playground-series-jan-2022) competition with weather [dataset](https://www.kaggle.com/adamwurdits/finland-norway-and-sweden-weather-data-20152019) as exogenous data
